### PR TITLE
[enhancement](profile) add instanceNum, tableIds to profile.

### PR DESCRIPTION
**Pull request by luozenglin**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.89 seconds
 load time: 502 seconds
 storage size: 17185028917 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221105115349_clickbench_pr_40520.html

File: be/src/vec/exec/scan/new_olap_scan_node.cpp [hunk 1]
```diff
@@ -132,6 +132,17 @@ static std::string olap_filters_to_string(const std::vector<doris::TCondition>&
...
 }
 
+static std::string tablets_id_to_string(
+        const std::vector<std::unique_ptr<TPaloScanRange>>& scan_ranges) {
+    std::stringstream ss;
+    ss << "[" << scan_ranges[0]->tablet_id;
+    for (int i = 1; i < scan_ranges.size(); ++i) {
+        ss << ", " << scan_ranges[i]->tablet_id;
+    }
+    ss << "]";
```
Comment (morningman) [inline be/src/vec/exec/scan/new_olap_scan_node.cpp:142]: the tablet id maybe too many?

File: fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java [hunk 2]
```diff
@@ -67,6 +67,11 @@ public class ProfileManager {
     public static final String DEFAULT_DB = "Default Db";
     public static final String SQL_STATEMENT = "Sql Statement";
     public static final String IS_CACHED = "Is Cached";
+
+    public static final String TOTAL_INSTANCES_NUM = "Total Instances Num";
+
+    public static final String INSTANCES_NUM_PER_FRAGMENT = "Instances Num Per Fragment";
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java:73]: Need backends list --> instance num

Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java]: done

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**